### PR TITLE
[chore](check) open shorten-64-to-32 error 

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

```
doris/be/src/pipeline/pipeline_fragment_context.cpp:1160:76: error: implicit conversion loses integer precision: 'size_type' (aka 'unsigned long') to 'int' [-Werror,-Wshorten-64-to-32]
 1159 |         _sink.reset(new MultiCastDataStreamSinkOperatorX(
      |                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 1160 |                 sink_id, sources, thrift_sink.multi_cast_stream_sink.sinks.size(), pool,
      |                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
```

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

File: be/src/pipeline/exec/scan_operator.cpp [hunk 1]
```diff
@@ -682,7 +684,7 @@ Status ScanLocalState<Derived>::_should_push_down_binary_predicate(
     DCHECK(constant_val->data == nullptr) << "constant_val should not have a value";
     const auto& children = fn_call->children();
     DCHECK(children.size() == 2);
-    for (size_t i = 0; i < children.size(); i++) {
```
Comment (HappenLee) [inline be/src/pipeline/exec/scan_operator.cpp:685]: why change the code？

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run cloud_p0

**Comment by Mryange** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**